kuibit/gw_utils.py

```diff
@@ -529,3 +529,43 @@ def signal_to_noise_ratio_from_strain(
     return np.sqrt(
         h_fft.inner_product(h_fft, noises=noise, fmin=fmin, fmax=fmax)
     )
+
+def effective_strain(h, *args, window_function=None, **kwargs):
+    r"""Return the effective strain
+
+    If ``window_function`` is not None, the window will be applied to the signal.
+    All the unknown arguments are passed to the window function.
+
+    This is a Kuibit version of h_eff from PostCactus by W. Kastaun.
+
+    :param h: Gravitational-wave strain.
+    :type h: :py:class:`~.TimeSeries`
+    :param window_function: If not None, apply ``window_function`` to the
+                            series before computing the strain.
+    :type window_function: callable, str, or None
+    :param args, kwargs: All the additional parameters are passed to
+                         the window function.
+
+    :returns: Effective Strain
+    :rtype: :py:class:`~.FrequencySeries`
+
+    """
+    from kuibit.frequencyseries import FrequencySeries
+
+    if not isinstance(h, ts.TimeSeries):
+        raise TypeError("Strain has to be a TimeSeries")
+    hp = h.real()
+    hc = -h.imag()
+    
+    # First, we window to avoid spectral leakage
+    if window_function is not None:
+        hp = hp.windowed(window_function, *args, **kwargs)
+        hc = hc.windowed(window_function, *args, **kwargs)
+
+    # Then, we take the Fourier transform
+    hp_fft = hp.to_FrequencySeries()
+    hc_fft = hc.to_FrequencySeries()
+
+    heff = hp_fft.amp**2. + hc_fft.amp**2.
+    heff = hp_fft.f * np.sqrt(heff / 2.)
+    return FrequencySeries(hp_fft.f, heff)
```

Comment:
This can be probably written as
```python
freqs = hp_fft.f
return freqs * np.sqrt((abs(hp_fft) + abs(hc_fft))/2)
```
which avoid calling `FrequencySeries` explicitly (so the import is not needed)

But, as I wrote elsewhere, if there's a way to avoid explicitly dealing with the real and imaginary part of the signal, that would be better.